src/forklift/models.py

```diff
@@ -53,3 +57,20 @@ def add_crates(self, crate_infos, defaults={}):
 
     def add_crate(self, crate_info):
         self.add_crates([crate_info])
+
+
+class Crate(object):
+    '''A module that defines a source and destination dataset that is a dependency of a pallet
+    '''
+
+    def __init__(self, source_name, source, destination, destination_name=None):
+        #: the name of the source data table
+        self.source_name = source_name
+        #: the name of the source database
+        self.source = source
+        #: the name of the destination database
+        self.destination = destination
+        #: the name of the output data table
+        self.destination_name = destination_name or source_name
+        #: the unique name of the crate
+        self.name = '{}::{}'.format(source, source_name)
```

Comment:
I think that this should be destination, shouldn't it? There are instances where the same source goes to two different destinations. We only want to skip if two crates are pointed at the same destination.